packages/shared/providers.yaml

```diff
@@ -2601,6 +2601,27 @@ fitbit:
         base_url: https://api.fitbit.com
     docs: https://docs.nango.dev/integrations/all/fitbit
 
+fortnox:
+    display_name: Fortnox
+    categories:
+        - accounting
+        - invoicing
+    auth_mode: OAUTH2
+    authorization_url: https://apps.fortnox.se/oauth-v1/auth
+    token_url: https://apps.fortnox.se/oauth-v1/token
```

Comment:
Pretty wild that their oauth2 flow is under a path named `oauth-v1`.

Reply:
yes it took me a while to stop debugging oauth v1 :D

Reply:
😮‍💨